.env

```diff
@@ -0,0 +1,9 @@
+APIBASEURL = https://api-f2f-cri-api.review-o.dev.account.gov.uk
```

Comment:
.env files shouldn't be committed it's in the .gitignore so I'm not even sure how this happened? Please can you remove and restore the .env.sample?

Reply:
will do now